### PR TITLE
partord 1.0.0

### PR DESCRIPTION
**Pull request by riccardo-bernardini**
Pull request for partord 1.0.0, a library for partial order sorting

### CODE REVIEW

File: index/pa/partord/partord-1.0.0.toml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name = "partord"
+description = "Small library for sorting with partial orders"
+version = "1.0.0"
+
+long-description = "This is a generic library that exports a sorting function that can operate with partial ordering, that is, it can happen that for some A /= B neither A < B nor A > B hold."
+
+website="https://gitlab.com/my-ada-library/partord/"
```
Comment (mosteo) [inline index/pa/partord/partord-1.0.0.toml:7]: I get a 404... Might the repository be set to private?

File: index/pa/partord/partord-1.0.0.toml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+authors = ["Riccardo Bernardini"]
+maintainers = ["Riccardo Bernardini <riccardo.bernardini@uniud.it>"]
+maintainers-logins = ["fintatarta"]
+licenses = "MIT"
+tags = ["sorting"]
+
+project-files = "partord.gpr"
```
Comment (mosteo) [inline index/pa/partord/partord-1.0.0.toml:15]: You don't need this line as the project file name matches the crate name.

**Comment by mosteo** [comment]
Hi Riccardo, thanks for your contribution. Once you fix the access issue with the repo, please ping me so I can relaunch the tests.

**Comment by riccardo-bernardini** [comment]
Yes, you was right, it was on private.

Now it should ork.

Thank you

--
Riccardo Bernardini
Tel          : +39-0432-55-8271
skype    :  bernardini.riccardo
LinkedIn : https://www.linkedin.com/in/riccardobernardini

________________________________________
Da: Alejandro R Mosteo ***@***.***>
Inviato: lunedì 31 maggio 2021 13:42
A: alire-project/alire-index
Cc: Riccardo Bernardini; Author
Oggetto: Re: [alire-project/alire-index] partord 1.0.0 (#282)

Hi Riccardo, thanks for your contribution. Once you fix the access issue with the repo, please ping me so I can relaunch the tests.

**Comment by mosteo** [comment]
Merged, thanks.

**Pull request merged**